DOCS-2305: Email corrections (#1306)
MultiSafepay/docs: content/payment-methods/prepaid-cards/paysafecard/activate-paysafecard.md

* If you do not fully use the value of a PIN when you pay, the residual credit will remain available for you to use later
* If you have not used up the balance in the first 12 months, then from the 13th month, you will be charged a monthly administration fee of 3 EUR of the residual balance.

> If you have any questions, email us at <sales@multisafepay.com>

Reviewer: ```suggestion
For any questions, email the Sales Team at <sales@multisafepay.com>
```
